assets/js/src/core/modules/asset/editor/image/tab-manager/tabs/properties/components/table/cells/select-cell/select-cell.tsx

```diff
@@ -0,0 +1,77 @@
+/**
+* Pimcore
+*
+* This source file is available under two different licenses:
+* - Pimcore Open Core License (POCL)
+* - Pimcore Commercial License (PCL)
+* Full copyright and license information is available in
+* LICENSE.md which is distributed with this source code.
+*
+*  @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+*  @license    https://github.com/pimcore/studio-ui-bundle/blob/1.x/LICENSE.md POCL and PCL
+*/
+
+import { type DefaultCellProps } from '@Pimcore/components/grid/columns/default-cell'
+import React, { useEffect, useRef, useState } from 'react'
+import { useEditMode } from '@Pimcore/components/grid/edit-mode/use-edit-mode'
+import { type RefSelectProps, Select } from 'antd'
+
+export const SelectCell = (props: DefaultCellProps): React.JSX.Element => {
+  const { isInEditMode, disableEditMode, fireOnUpdateCellDataEvent } = useEditMode(props)
+  const [open, setOpen] = useState<boolean>(false)
+  const selectRef = useRef<RefSelectProps>(null)
+
+  useEffect(() => {
+    if (isInEditMode) {
+      setOpen(true)
+      selectRef.current?.focus()
+    }
+  }, [isInEditMode])
+
+  function saveValue (value: string): void {
+    fireOnUpdateCellDataEvent(value)
+    disableEditMode()
+  }
+
+  function getCellContent (): React.JSX.Element {
+    if (!isInEditMode) {
+      return (
+        <>
+          { props.getValue() }
+        </>
+      )
+    }
+
+    function onBlur (e: React.FocusEvent<HTMLInputElement>): void {
+      saveValue(e.target.value)
+    }
+
+    function onKeyDown (e: React.KeyboardEvent<HTMLInputElement>): void {
+      if (e.key === 'Escape') {
+        disableEditMode()
+      }
+    }
+
+    return (
+      <Select
+        defaultValue={ props.getValue() }
+        onBlur={ onBlur }
+        onChange={ saveValue }
+        onKeyDown={ onKeyDown }
+        open={ open }
+        options={ [
+          { value: 'jack', label: 'Jack' },
+          { value: 'lucy', label: 'Lucy' },
+          { value: 'Yiminghe', label: 'yiminghe' }
+        ] }
```

Comment:
Is it right that this is still hardcoded?
What are we missing here?

Reply:
I was thinking about the translation of the stings. 
We only get a comma-separated list from the BE, something like "foo,bar,bazz." 
Wasn't sure how I should handle that - I implemented a basic display of the predefined options for now. (see https://github.com/pimcore/studio-ui-bundle/pull/324/commits/24cc73433484e68ddd6653de91fcacacabf06e2d)

@markus-moser, does Pimcore have an option to translate the options of a select in the predefined props?

Reply:
@Corepex In the classic UI it's not translatable. So let's leave it as not translateable for now.

Reply:
Thanks for checking @markus-moser. I guess than the current implementation should do it :)